modules/nw-sriov-hwol-using-vf-management-port.adoc

```diff
@@ -0,0 +1,122 @@
+// Module included in the following assemblies:
+//
+// * networking/hardware_networks/configuring-hardware-offloading.adoc
+
+:_content-type: PROCEDURE
+[id="using-vf-improve-network-traffic-performance_{context}"]
+= Using a virtual function to improve network traffic performance
+
+Follow this procedure to assign a virtual function to the OVN-Kubernetes management port and increase its network traffic performance.
+
+This procedure results in the creation of two pools: the first has a virtual function used by OVN-Kubernetes, and the second comprises the remaining virtual functions.
+
+.Prerequisites
+
+* You installed the OpenShift CLI (`oc`).
+* You have access to the cluster as a user with the `cluster-admin` role.
+
+.Procedure
+
+. Add the `network.operator.openshift.io/smart-nic` label to each worker node with a SmartNIC present by running the following command:
++
+[source,terminal]
+----
+$ oc label node <node-name> network.operator.openshift.io/smart-nic=
+----
++
+Use the `oc get nodes` command to get a list of the available nodes.
+
+. Create a policy named `sriov-node-policy.yaml` with content such as the following example:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: sriov-node-policy
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: netdevice
+  eSwitchMode: "switchdev"
+  nicSelector:
+    deviceID: "1019"
+    rootDevices:
+    - 0000:d8:00.0
+    vendor: "15b3"
+    pfNames:
+    - ens8f0#1-5 <.>
+  nodeSelector:
+    network.operator.openshift.io/smart-nic: ""
+  numVfs: 6 <.>
+  priority: 5
+  resourceName: mlxnics
+----
+<.> Replace this device with the appropriate network device for your use case.
+<.> The value provided here is an example. Replace this value with one that meets your requirements. For more information, see _SR-IOV network node configuration object_ in the _Additional resources_ section.
+
+. Create a policy named `sriov-node-mgmt-vf-policy.yaml` for the management port with content such as the following example:
++
+[source,yaml]
+----
+apiVersion: sriovnetwork.openshift.io/v1
+kind: SriovNetworkNodePolicy
+metadata:
+  name: sriov-node-mgmt-vf-policy
+  namespace: openshift-sriov-network-operator
+spec:
+  deviceType: netdevice
+  eSwitchMode: "switchdev"
+  nicSelector:
+    deviceID: "1019"
+    rootDevices:
+    - 0000:d8:00.0
+    vendor: "15b3"
+    pfNames:
+    - ens8f0#0-0 <.>
```

Comment:
For me, it seems more logical to put this yaml first (this one is vf 0, i.e. the first vf).